vagrant/build.sh

```diff
@@ -30,6 +30,14 @@ DEBUG=1 DEBUG2=1 ./build.sh
 go get github.com/armPelionEdge/maestro-shell || true
 go install github.com/armPelionEdge/maestro-shell
 
+# Generate identity certificates to use to connect to Pelion
+cd ..
+git clone git@github.com:armPelionEdge/edge-utils.git
+cd edge-utils/debug_scripts/get_new_gw_identity/developer_gateway_identity
+chmod +x generate_self_signed_certs.sh
+# TODO swap out ACCOUNTID and DEVICEID with real IDs, not fake ones
+OU=ACCOUNTID internalid=DEVICEID ./generate_self_signed_certs.sh $MAESTRO_CERTS
+
```

Comment:
Making sure, this changes the active directory for the following `go get` command.
Is this intended?

Reply:
Not intended, but shouldn't be a problem because go-get installs into GOPATH.  Still, @mray190,  it would be nice if this block were more self-contained with a cd $SOME_SOURCE_DIR on line 34 rather than relying on ".." which is fragile.